### ISSUE TITLE
how to pass function inside of parameter

### ISSUE THREAD

**Issue by hbakhtiyor** [nelmio/alice#292]
e.g.
```yaml
    parameters:
         address_one: bar
         address_two: foo
         address_three: fizz
         ....
 
    user_{one, two, three, ....}:
        name: bob
        address:  <{address_<current()>}>
```

want to pass like this `<{address_one}>, <{address_two}>, ...` inside of the array

how to achieve this without any duplication?

**Comment by petegore** [comment]
I'm interested in that to, it can be useful in a random ManyToOne relation too : 

```yaml
group1 :
    users : <ArrayCollection(<numberBetween(1,10)>x @user*)> 
```

**Comment by theofidry** [comment]
Not sure you can do it, but you could solve your problem by using a custom faker provider :)

**Comment by theofidry** [comment]
Reviewing this after taking a closer look in the code, this indeed won't work as right now `<{...}>` delimitates a parameter name and is not evaluated. As a result `<{address_<current()>}>` refers to the parameter `address_<current()>`.

Will consider that for v3.

**Comment by theofidry** [comment]
Closed in favour of #354
